src/software/simulation/physics/physics_ball_test.cpp

```diff
@@ -407,23 +387,27 @@ TEST(PhysicsBallTest, test_chip_ball_with_collisions)
     obstacle_body->CreateFixture(&obstacle_fixture_def);
 
     EXPECT_FALSE(physics_ball.isInFlight());
-    physics_ball.chip(Vector(2, 0));
+    physics_ball.setInFlightForDistance(2.0f);
     EXPECT_TRUE(physics_ball.isInFlight());
 
-    for (unsigned int i = 0; i < 100; i++)
+    for (unsigned int i = 0; i < 300; i++)
```

Comment:
What's the reason for changing this?

Reply:
Ball is now moving at 1m/s, and we want to make sure it passes all the way through the object at (2, 0), so this just makes sure there's enough time for that to happen